aqt/installer.py

```diff
@@ -533,9 +603,13 @@ def run_list_qt(self, args: argparse.ArgumentParser):
         if args.target not in ArchiveId.TARGETS_FOR_HOST[args.host]:
             raise CliInputError("'{0.target}' is not a valid target for host '{0.host}'".format(args))
         if args.modules:
-            modules_ver, modules_query, is_long = args.modules[0], tuple(args.modules), False
+            assert len(args.modules) == 2, "broken argument parser for list-qt"
+            modules_query = MetadataFactory.ModulesQuery(args.modules[0], args.modules[1])
+            modules_ver, is_long = args.modules[0], False
         elif args.long_modules:
-            modules_ver, modules_query, is_long = args.long_modules[0], tuple(args.long_modules), True
+            assert len(args.modules) == 2, "broken argument parser for list-qt"
+            modules_query = MetadataFactory.ModulesQuery(args.modules[0], args.modules[1])
+            modules_ver, is_long = args.long_modules[0], True
```

Comment:
Why this refers "args.modules" not "args.long_modules" ?
```suggestion
            assert len(args.long_modules) == 2, "broken argument parser for list-qt"
            modules_query = MetadataFactory.ModulesQuery(args.long_modules[0], args.long_modules[1])
            modules_ver, is_long = args.long_modules[0], True
```

Reply:
Good catch. I'm alarmed that the test suite did not catch this though; maybe we need to add a test for this.